files/en-us/web/css/color_value/rgba()/index.html

```diff
@@ -22,7 +22,7 @@ <h4>Note</h4>
 <h2 id="Syntax">Syntax</h2>
 
 <pre class="brush: css">rgba(255,255,255,.5) /* white with 50% opacity */
-rgba(255 255 255,.5); /* CSS Colors 4 space-separated values */
+rgba(255 255 255/.5); /* CSS Colors 4 space-separated values */
```

Comment:
```suggestion
rgba(255 255 255 / 0.5); /* CSS Colors 4 space-separated values */
```

I'd like to see spaces between the slash and a leading zero here.